app/assets/stylesheets/components/_footer.sass

```diff
@@ -0,0 +1,11 @@
+$Footer-white: Theme-color(background, footer)
```

Comment:
This [one](http://csswizardry.com/2014/03/naming-ui-components-in-oocss/) as well. That's just to give you awareness of this sort of problems. It's not supposed to be a rule to follow, since that's just a suggestion that fits his needs